ibtk/src/lagrangian/FEDataManager.cpp

```diff
@@ -1180,8 +1181,11 @@ FEDataManager::interpWeighted(const int f_data_idx,
         TBOX_ASSERT(X_dof_map.variable_order(d) == X_order);
     }
 
+    // convenience alias for the quadrature key type used by FECache and FEMapCache
+    using quad_key_type = std::tuple<libMesh::ElemType, libMesh::QuadratureType, libMesh::Order>;
     FECache F_fe_cache(dim, F_fe_type);
     FECache X_fe_cache(dim, X_fe_type);
+    FEMapCache F_fe_map_cache(dim);
```

Comment:
There is no need to identify the cached `FEMap` as associated with the `F` variable --- it probably is clearer to call it `fe_map_cache`, because it is independent of the FE space used for `F`.